### PR TITLE
remove rebuilds and buildconfigs

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/plugins/exit_sendmail.py [hunk 1]
```diff
@@ -53,7 +52,7 @@ class SendMailPlugin(ExitPlugin):
         "exit_plugins": [{
                 "name": "sendmail",
                 "args": {
-                    "send_on": ["auto_canceled", "auto_fail"],
+                    "send_on": ["manual_fail", "manual_canceled"],
```
Comment (MartinBasti) [inline atomic_reactor/plugins/exit_sendmail.py:55]: is this correct?

Comment (MartinBasti) [reply atomic_reactor/plugins/exit_sendmail.py:55]: Ah nevermind, docstring :D

**Review by MartinBasti — APPROVED** [review]

**Comment by rcerven** [comment]
tests are failing because coverall is down

**Comment by MartinBasti** [comment]
Can we have description in commit message why pls?

File: atomic_reactor/source.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __init__(self, build_path):
                 raise
 
         self.release_env_var = self.data.get('set_release_env')
-        self.autorebuild = self.data.get('autorebuild') or {}
+        self.autorebuild = {}
```
Comment (chmeliik) [inline atomic_reactor/source.py:56]: Is this still needed?

**Review by lkolacek — APPROVED** [review]

**Review by chmeliik — COMMENTED** [review]
LGTM, just not sure if the autorebuild param in SourceConfig is still needed

**Comment by rcerven** [comment]
> Can we have description in commit message why pls?

added

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Comment by rcerven** [comment]
rn updated

**Pull request merged**